dev-haskell/beam-automigrate/beam-automigrate-0.1.2.0.ebuild

```diff
@@ -0,0 +1,55 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+# ebuild generated by hackport 0.6.7.9999
+
+CABAL_FEATURES="lib profile haddock hoogle hscolour test-suite"
+inherit haskell-cabal
+
+DESCRIPTION="DB migration library for beam, targeting Postgres"
+HOMEPAGE="https://hackage.haskell.org/package/beam-automigrate"
+SRC_URI="https://hackage.haskell.org/package/${P}/${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0/${PV}"
+KEYWORDS="~amd64 ~x86"
+IUSE="ghcipretty integration-tests werror examples"
+
+PATCHES=( "${FILESDIR}/${P}-add-examples-flag.patch" )
+
+RDEPEND=">=dev-haskell/aeson-1.4.4:=[profile?] <dev-haskell/aeson-1.5:=[profile?]
+	>=dev-haskell/beam-core-0.9:=[profile?] <dev-haskell/beam-core-0.10:=[profile?]
+	>=dev-haskell/beam-postgres-0.5:=[profile?] <dev-haskell/beam-postgres-0.6:=[profile?]
+	>=dev-haskell/dlist-0.8.0:=[profile?] <dev-haskell/dlist-0.10:=[profile?]
+	>=dev-haskell/microlens-0.4.10:=[profile?] <dev-haskell/microlens-0.6:=[profile?]
+	>=dev-haskell/mtl-2.2.2:=[profile?] <dev-haskell/mtl-2.4:=[profile?]
+	>=dev-haskell/postgresql-simple-0.5.4:=[profile?] <dev-haskell/postgresql-simple-0.7.0.0:=[profile?]
+	>=dev-haskell/pretty-simple-2.2.0:=[profile?] <dev-haskell/pretty-simple-3.3:=[profile?]
+	>=dev-haskell/quickcheck-2.13:2=[profile?] <dev-haskell/quickcheck-2.15:2=[profile?]
+	>=dev-haskell/quickcheck-instances-0.3:=[profile?] <dev-haskell/quickcheck-instances-0.4:=[profile?]
+	>=dev-haskell/scientific-0.3.6:=[profile?] <dev-haskell/scientific-0.5:=[profile?]
+	>=dev-haskell/splitmix-0.0.3:=[profile?] <dev-haskell/splitmix-0.1:=[profile?]
```

Comment:
Same here, there is a `splitmix` newer than this available, is it possible for you to test?